### ISSUE TITLE
<Branch for Oct. 20th #46>If JSON is incorrect SAS test harness should not start the server

### ISSUE THREAD

**Issue by pawan25062** [Wireless-Innovation-Forum/Citizens-Broadband-Radio-Service-Device#48]
[WINNF.FT.C.REG.3_2017-10-26T06.56.57Z.zip](https://github.com/Wireless-Innovation-Forum/Citizens-Broadband-Radio-Service-Device/files/1417707/WINNF.FT.C.REG.3_2017-10-26T06.56.57Z.zip)
While running JSON file with airInterface parameter in it, the test case is getting failed
2017-10-26T06:57:10.982Z - INFO - engine sent successfully, the response to CBRS  : "Expecting property name: line 32 column 7 (char 1243)"
2017-10-26T06:57:11.684Z - INFO - The final result of the test : WINNF.FT.C.REG.3 is - FAIL

		>>> Now here issue is, SAS test harness should check whether JSON is correct or not before starting the server. For this branch SAS test harness sending this error as registration response which should not happen. If JSON is incorrect SAS test harness should not start the server.

**Comment by cwilliams-ericsson** [comment]
To be clear:  bad format in WINNF.FT.C.REG.3 test case (as shown below) is root cause (lack of open quotation mark around "radioTechnology".  I view this as low priority, although it would be a useful enhancement.

			"registrationRequest":
        	[
            	{
 				  "userId":"${maximumLength:128}", 
 				  "fccId":"${maximumLength:128}", 				  

				  "measCapability": {"$or":[["EUTRA_CARRIER_RSSI_NON_TX"],["EUTRA_CARRIER_RSSI_ALWAYS"],["EUTRA_CARRIER_RSSI_NON_TX", "EUTRA_CARRIER_RSSI_ALWAYS"] ]},
				  "cbsdCategory":{"$or":["A","B"]},
				  "airInterface":
					{
						radioTechnology";("$or":["E_UTRA","CAMBIUM_NETWORKS"]),
						},
